SECURITY.md

```diff
@@ -1,27 +1,77 @@
+
+    ```
+    Example: We have a vulnerable package foobar@1.x.y and 1.x.z is the fix. In package.json, we have foobar@^1.x.y which should update you to all future minor/patch versions, without incrementing the major version.
+
+    Find foobar@1.x.y in yarn.lock and remove it.
+    Run yarn osd bootstrap
+
+    ```
+
+    - If package.json is not correct, we can add version in the resolutions [section](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) for all the package sub-deps or specific package sub-dep. For more on version updates please see [Why](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-why-would-you-want-to-do-this) and [How](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it) to upgrade.
+
+    ```
+    Example: If foobar@1.x exists for subdeps in yarn.lock file. Then edit the package.json file and add **/foobar@1.y in resolution section as shown below to enforce the 1.y.
+
+    Add the package to resolution in package.json:
+    'resolutions': { "**/foobar": "^1.y",
+                     "**/foo": "^2.x" ,
+                     "**/bar": "^3.k"}
+    Then run: yarn install for updating yarn.lock file
+
+    ```
+
+    Please be aware of that fixing nested dependency can be tricky. Sometimes, bump the dependent package will auto bump all the nested dependencies which will solve multiple security vulnerabilities and provide a more maintainable code base.
+
+- To backport the CVEs fix to previous versions, add backport label (ex: `backport 1.x`) to trigger auto backport. If auto backport fail, pls use the following steps to manually backport:
+
+    - Identify the pull request you want to backport and the target backport version.
+    - Create a new local branch from the target version.
+    - Cherry-pick the changes from the pull request into the new branch. To do this, you can use the `git cherry-pick` command followed by the hash of the pull request commit. For example: `git cherry-pick 123456`.
+    - Resolve any conflicts. This step may require some manual intervention.
+    - Test the changes thoroughly.
+    - Push the new branch to the appropriate remote repository.
+    - Submit a new pull request to the target version for the backported changes.
+
+    ```
+       Example: backport a pull request with hash 123 in main to 1.3
+
+       1. Fetch the latest changes from upstream or whatever you name the remote upstream repo:
+       git pull upstream
+
+       2. Create a new local branch from the target version:
+       git checkout -b backport-cve upstream/1.3
+
+       3. Cherry pick the changes:
+       git cherry-pick 123
+
+       4. Resolve any conflicts.
+
+       5. Push to your origin forked repo:
+       git push -u origin backport-cve
+
+       6. Submit a new pull request to 1.3.
```

Comment:
```suggestion

```
These instructions could change; it would be best to tell users to follow the steps that would be provided to them in the PR.

Reply:
I will mention these steps are just for reference.